rest-api-spec/src/yamlRestCompatTest/resources/rest-api-spec/test/v7compat/nodes.hotthreads/10_basic_compat.yml

```diff
@@ -0,0 +1,21 @@
+---
+setup:
+  - skip:
+      version: "9.0.0 - "
+      reason: "compatible from 8.x to 7.x"
+      features:
+        - "headers"
+        - "warnings_regex"
+
+---
+"Get hot threads":
+
+  - do:
+      headers:
+        Content-Type: "application/vnd.elasticsearch+json;compatible-with=7"
+        Accept: "application/vnd.elasticsearch+json;compatible-with=7"
+      nodes.hotthreads: {}
+      warnings_regex:
+        - ".*hot_?threads.* is a deprecated endpoint.*"
+  - match:
+      $body: /:::/
```

Comment:
This `match` is not my most favorite code ever. I wanted to verify that the response actually resembles a `hot_threads` response, but the regex match has to be on the first line and there's not all that much for us to match on there:

```
::: {runTask-0}{rNO3-C-BStGLmQ_xUdIecA}{PiMCJKfOSnuRkp0Uk1E7ig}{127.0.0.1}{127.0.0.1:9300}{cdfhilmrstw}{ml.machine_memory=68719476736, xpack.installed=true, testattr=test, ml.max_open_jobs=512, ml.max_jvm_size=536870912}
   Hot threads at 2021-06-07T15:44:46.079Z, interval=500ms, busiestThreads=3, ignoreIdleThreads=true:
```

Reply:
Anyway, there is a `:::` so I'm matching on that.